cmake/toolchain/gcc.sve512.cmake

```diff
@@ -0,0 +1,13 @@
+##==================================================================================================
+##  EVE - Expressive Vector Engine
+##  Copyright : EVE Contributors & Maintainers
+##  SPDX-License-Identifier: MIT
+##==================================================================================================
+set(CMAKE_SYSTEM_NAME Linux     )
+set(CMAKE_SYSTEM_PROCESSOR arm  )
+
+set(CMAKE_C_COMPILER    aarch64-linux-gnu-gcc-11  )
+set(CMAKE_CXX_COMPILER  aarch64-linux-gnu-g++-11  )
+set(CMAKE_CXX_FLAGS     "-Wno-psabi -DEVE_NO_FORCEINLINE ${EVE_OPTIONS} -march=armv8-a+sve -msve-vector-bits=512" )
+
+set(CMAKE_CROSSCOMPILING_CMD qemu-aarch64)
```

Comment:
All 3 running with the same emulator is weird - how does qemu know the instruction width.

Reply:
THis needs investigation yes